dmsite/backend/dmsite/db_manager/db_manager.py

```diff
@@ -179,41 +155,88 @@ def check_db():
         return -1, {"error": ERR_STR, "errorStr": e.__str__()}
 
 
-def make_query(table, index, value, compare):
-    if db_status is False:
+def scan(table, attrib, comp, db=None):
+    if db_status is False and db is None:
+        status, resp = check_db();
+        if status is -1:
+            return resp
+    try:
+        if db is not None:
+            files = db.Table(table)
+        else:
+            files = dynamodb.Table(table)
+        response = files.scan(
```

Comment:
Scan has a limit of 1mb worth of items. Will that be a concern? https://stackoverflow.com/questions/43199385/how-to-exceed-the-limit-of-scan-data-for-1mb-in-dynamodb

Reply:
Most of the tables are going to be relatively small - it should be fine.